src/plugins/vis_type_timeseries/server/lib/vis_data/helpers/bucket_transform.js

```diff
@@ -161,19 +161,24 @@ export const bucketTransform = {
     };
   },
 
-  derivative: (bucket, metrics, bucketSize) => {
+  derivative: (bucket, metrics, intervalString) => {
     checkMetric(bucket, ['type', 'field']);
+
     const body = {
       derivative: {
         buckets_path: getBucketsPath(bucket.field, metrics),
         gap_policy: 'skip', // seems sane
-        unit: bucketSize,
+        unit: intervalString,
```

Comment:
according to the documentation there should be a string value here, not the size of the bucket

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-derivative-aggregation.html#_units